docs/guides/room/receive.mdx

```diff
@@ -277,7 +264,7 @@ class _ParticipantState extends State<ParticipantWidget> {
 
 ## Speaker detection
 
-When audio tracks are published, LiveKit will automatically detect participants that are speaking. Speaker updates are sent for both local and remote participants. They will fire on both Room and each participant.
+When audio tracks are published, LiveKit will detect participants that are speaking. Speaker updates are sent for both local and remote participants. They will fire on both Room and each participant.
```

Comment:
s/that are speaking/whom are speaking

They will fire on both, Room and Participant objects.